includes/polling/os/quantastor.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
This needs to be ported to the new OS format, see https://github.com/librenms/librenms/blob/master/LibreNMS/OS/Qnap.php for example.